packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -350,11 +497,28 @@ function handler(app: Application) {
       await handler.cleanDatastoreTable(watchedPRs, app, context);
       return;
     }
+
+    if (context.payload.find_hanging_prs === true) {
+      logger.info('Entering job to pick up any hanging PRs');
+      // we cannot search in an org without the bot installation ID, so we need
+      // to divide up the cron jobs based on org
+      if (context.payload.org === 'googleapis') {
+        await handler.scanForMissingPullRequests(context.github, 'googleapis');
```

Comment:
rather than using an `if` statement here, you could just do:

`await handler.scanForMissingPullRequests(context.github, context.payload.org);`

Reply:
We could... we're just going to end up with a lot of errors in the logs and additional API calls that we don't need to make. I'm not sure what is better here, less lines of code and more logs or vice versa!

Reply:
what I"m getting at is that `context.payload.org` is going to be one of `googleapis` or `GoogleCloudPlatform` I believe?

---

Why do we return early for `GoogleCloudPlatform` but not `googleapis`, out of curiosity?